Fix styles on table and figure

oliverguenther opened the pull request:
https://github.com/opf/commonmark-ckeditor-build/pull/50
https://community.openproject.com/wp/33349

oliverguenther committed:
Fix styles on table and figure

HDinger reviewed (changes requested):
There are some issues with the height calculation:

**Having multiple tables below each other**
I have two tables below each other, both have a set height. This is how it looks like in the edit mode:
<img width="1087" alt="Bildschirmfoto 2020-05-25 um 15 52 10" src="https://user-images.githubusercontent.com/7457313/82818938-cf1d1600-9e9f-11ea-916d-9873720db1ac.png">

and this is how it looks like after saving (note, that the bottom table is cut off and text is missing):
<img width="1092" alt="Bildschirmfoto 2020-05-25 um 15 52 20" src="https://user-images.githubusercontent.com/7457313/82818960-d9d7ab00-9e9f-11ea-804f-aa7edfc211ac.png">

**I can write next to the table**
When I have a table with a set height, I can write next to the table. (Select the table and then press Enter)
<img width="1018" alt="Bildschirmfoto 2020-05-25 um 15 57 01" src="https://user-images.githubusercontent.com/7457313/82819249-61bdb500-9ea0-11ea-803a-ec1e25da5d34.png">
This is especially problematic if I set the table orientation to center, since then the text is behind the table and not visible any more.

In general I am wondering whether we should disable the table orientation feature? I assumed for quite some time, that it changes the text orientation within the cells and not the table orientation with its text itself. I first thought, it was a bug and I guess, Robin stumbled over that, too. I think users are confused, and I don't see a big benefit, of moving the table to the right. 

**One additional note**
Like Robin already mentioned, I think that is is confusing, that the table cell background is not changed in edit mode, when I set it for the whole table. However, this might be solved separately.

oliverguenther commented:
I will look at the first problem. the other two look like. ckeditor issues that we cannot easily fix - i.e. we cannot disable parts of the styling dialog

oliverguenther committed:
Remove specific styling for editable

The styles are being overwritten and cause the table cell background not
be visible in editing
